app/static/js/base.js

```diff
@@ -1,9 +1,43 @@
-$('.attack-button').on('click', function() {
-    $(this).closest('div.table-parent').find('.defense').hide();
-    $(this).closest('div.table-parent').find('.attack').show();
-});
-
-$('.defense-button').on('click', function() {
-    $(this).closest('div.table-parent').find('.attack').hide();
-    $(this).closest('div.table-parent').find('.defense').show();
-});
\ No newline at end of file
+;(function($){
+    'use strict';
+
+    function setUserResources($btn, data, $userResources) {

```

Comment:
```suggestion
    function setUserResources($btn, { added_resource, turns }, $userResources) {
```

Reply:
i didn't know you can do that, this is great, thanks